src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface TodoModalProps {
+  todo: Todo;
+  /* eslint-disable */
+
+  users: User[];
+  onCloseTodoModal: () => void;
+}
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId?: number;
+}
+
+interface User {
+  id: number;
+  name: string;
+  email: string;
+}
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  todo,
+  onCloseTodoModal,
+}) => {
+  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(true);
```